src/app/language-support/page.tsx

```diff
@@ -29,6 +41,47 @@ export default function Page() {
     return 'null';
   };
 
+  const isCaseInterpretation = (obj: AllLanguageSupport) =>
```

Comment:
Do you think we can update this so we check using the `listing_type` fields instead? I don't think the `CaseListing` schema currently has this field, so we'll need to add it! 

could we add a `listing_type` field to the schema? (this would be defaulted to "Case" in the `schema.ts` file). Once you pull the data for case-specific interpretations, we would want to change this field to "Case Interpretation".

Reply:
we would need to add the `listing_type` field when pulling the data using with queries. (i.e. add the value "Case" for the `listing_type` field in the `getAllCases` query, and add the value "Case Interpretation" for the `listing_type` field in the `/queries/LanguageSupport.ts` query pulling case-specific interpretation)

Reply:
done!

Reply:
Actually I take this back, after further discussion with @jinkang-0 (huge ++), this implementation would error because the type would be mismatched if we change to "Case Interpretation" since we have `listing_type: 'Case';` for `CaseListing`. Moving forward this is how we'll distinguish the types instead:
<img width="422" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/7d0321dc-7435-4dc8-a6d4-f5884b8b8f8e">